app/src/main/java/com/p2p/presentation/home/games/GamesViewModel.kt

```diff
@@ -35,7 +35,7 @@ class GamesViewModel(private val userSession: UserSession) : BaseViewModel<Unit>
     /** Open the view that corresponds to create the [selectedGame]. */
     fun createGame(userName: String?) {
         if (!validateAndSaveName(userName)) return
-        // TODO: perform actions for createGame (create an event GoToCreateGame and dispatch it)
+        dispatchSingleTimeEvent(GoToCreateTuttiFrutti)
```

Comment:
Acá está lo que te diste cuenta el otro día 😅 
```suggestion
        when(selectedGame) {
            TUTTI_FRUTTI -> dispatchSingleTimeEvent(GoToCreateTuttiFrutti)
        }
```